core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -647,4 +647,42 @@ package object config {
       .stringConf
       .toSequence
       .createWithDefault(Nil)
+
+  private[spark] val KAFKA_DELEGATION_TOKEN_ENABLED =
+    ConfigBuilder("spark.kafka.delegation.token.enabled")
+      .doc("Set to 'true' for obtaining delegation token from kafka.")
+      .booleanConf
+      .createWithDefault(false)
+
+  private[spark] val KAFKA_BOOTSTRAP_SERVERS =
+    ConfigBuilder("spark.kafka.bootstrap.servers")
+      .doc("A list of coma separated host/port pairs to use for establishing the initial " +
+        "connection to the Kafka cluster. For further details please see kafka documentation.")
+      .stringConf.createOptional
```

Comment:
One method call per line (a.k.a. "follow the pattern").

Reply:
Fixed (a.k.a. "followed another existing pattern").